helm/k8s-initiator-app/Chart.yaml

```diff
@@ -7,4 +7,5 @@ icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
 name: k8s-initiator-app
 sources:
 - https://github.com/giantswarm/k8s-initiator-app
+- https://github.com/giantswarm/k8s-initiator-app/blob/master/README.md
```

Comment:
```suggestion
- https://raw.githubusercontent.com/giantswarm/k8s-initiator-app/v[[ .Version ]]/README.md
```

Reply:
If all goes we'll we'd end up with a URL like: https://raw.githubusercontent.com/giantswarm/k8s-initiator-app/v0.9.1/README.md which goes to real raw markdown, instead of page that actually has a bunch of extra html and stuff.